tests/repr_test.py

```diff
@@ -73,9 +73,10 @@ def test_repr_from_pandas():
     dd_dict = {
         'boolean': [True, True, False, None, True],
         'text': ['This', 'is', 'some', 'text', 'so...'],
-        'numbers_1': [1, 30, -2, 1.5, 0.000],
-        'numbers_2': [1, None, -2, 1.5, 0.000],
-        'numbers_3': [1, np.nan, -2, 1.5, 0.000],
+        'text_missing': pd.Series(['Some', 'parts', None, 'missing', None], dtype='string'),
+        'float': [1, 30, -2, 1.5, 0.000],
+        'float_missing': [1, None, -2, 1.5, 0.000],
+        'int_missing': pd.Series([1, None, 5, 1, 10], dtype='Int64'),
```

Comment:
I don't think this should go into repr_test. Internally, vaex should never see these types, this should be handled at a higher level. It would make sense to have `from_pandas_test.py` and check if vaex indeed converts the types.